decidim-comments/README.md

```diff
@@ -35,6 +35,38 @@ And then execute:
 $ bundle
 ```
 
+## How to contribute
```

Comment:
Could you also write a quick section explaining what's the use of this extension?